app/services/platform/user_filestore_adapter.rb

```diff
@@ -1,13 +1,32 @@
 module Platform
   class UserFilestoreAdapter
-    def initialize(session, root_url: ENV['FILESTORE_URL'], service_slug: ENV['SERVICE_SLUG'], payload:)
+    include Platform::Connection
+    attr_reader :session, :root_url, :service_slug, :payload
+
+    def initialize(
+      session,
+      root_url: ENV['FILESTORE_URL'],
+      service_slug: ENV['SERVICE_SLUG'],
+      payload:
+    )
       @session = session
       @root_url = root_url
       @service_slug = service_slug
       @payload = payload
     end
 
-    def upload(params)
+    def call
+      return if root_url.blank?
+
+      # use the subject as the same as user_id which is on the url
```

Comment:
can we remove these too?

Reply:
Neither this ones

Reply:
oh yes you're right, I was looking at this commit by commit rather than all the files 😆